plugins/ltac2/tac2stdlib.ml

```diff
@@ -618,6 +618,20 @@ let () =
 
 (** Tactics around Evarconv unification (in [Ltac2/Unification.v]). *)
 
+let to_conv_pbv v = match Tac2ffi.to_int v with
+| 0 -> Conversion.CONV
+| 1 -> Conversion.CUMUL
+| _ -> assert false
+
+let conv_pb = Tac2ffi.make_repr (fun _ -> assert false) to_conv_pbv
+
+let () =
+  define "infer_conv" (conv_pb @-> transparent_state @-> constr @-> constr @-> tac bool) @@ fun pb ts c1 c2 ->
+  Tac2core.pf_apply @@ fun env sigma ->
+  match (Reductionops.infer_conv ~pb:pb ~ts:ts env sigma c1 c2) with
```

Comment:
```suggestion
  match Reductionops.infer_conv ~pb ~ts env sigma c1 c2 with
```

Reply:
@SkySkimmer Isn't that needed to overwrite the default value of ~pb and ~ts to the variables pb and ts

Reply:
???
in ocaml `~x` is completely equivalent to `~x:x`